src/pairwise.jl

```diff
@@ -149,14 +160,14 @@ function promote_type_union(::Type{T}) where T
 end
 
 function _pairwise(::Val{skipmissing}, f, x, y, symmetric::Bool) where {skipmissing}
-    x′ = x isa Union{AbstractArray, Tuple, NamedTuple} ? x : collect(x)
-    y′ = y isa Union{AbstractArray, Tuple, NamedTuple} ? y : collect(y)
+    x′ = x isa Union{AbstractArray,Tuple,NamedTuple} ? x : collect(x)
+    y′ = y isa Union{AbstractArray,Tuple,NamedTuple} ? y : collect(y)
```

Comment:
```suggestion
    x′ = x isa Union{AbstractArray, Tuple, NamedTuple} ? x : collect(x)
    y′ = y isa Union{AbstractArray, Tuple, NamedTuple} ? y : collect(y)
```

Reply:
A result of my preference for VSCode "Format Document" - see remarks above.

Reply:
The situation I have in mind is when `f` is expensive to calculate (as is the case for `corkendall` and `corspearman`) and so the important thing is to ensure that each thread makes the same number of calls to `f`, so as to have all threads finish their work near simultaneously. I was able to see the benefit of `equal_sum_subsets`  when calculating a 5,000 x 5,000 kendall correlation matrices simply by observing the loading of cores in Windows task manager. My prior approach had a nasty tail of only one or two cores doing any work. `equal_sum_subsets` solved that with about a 25% speedup (if I recall correctly).

I also experimented with with using [ChunkSplitters](https://github.com/JuliaFolds2/ChunkSplitters.jl) with `split = : scatter` but that approach does not do quite as good a job of equal division between threads of calls to `f`. 

Surely considerations of memory layout would only come into play if `f` is a _really_ fast function? `LinearAlgebra.dot` might be a good example and I was very happy with the performance in that case.

In conclusion I doubt that there exists a significantly faster approach.